nntrainer/src/parse_util.cpp

```diff
@@ -279,13 +281,11 @@ unsigned int parseLayerProperty(std::string property) {
       return (i);
     }
   }
-  ret = i - 1;
 
-  return ret;
+  return property_string.size() - 1;
```

Comment:
I think your proposal can be applied here also.

Reply:
Updated. I missed it.